src/vaadin-contextmenu-items-mixin.html

```diff
@@ -156,6 +156,12 @@
       // Store the reference to align based on parent overlay coordinates
       subMenu.$.overlay._setParentOverlay(parent);
 
+      // Set theme attribute from parent element
+      const parentTheme = parent.getAttribute('theme');
+      if (parentTheme && parentTheme !== '') {
```

Comment:
`if (parentTheme)` should cover both cases ('' is also falsy)

Reply:
Updated.